src/main/waveform/waveform.core.js

```diff
@@ -66,7 +66,7 @@
    * data) or 'json'
    * @param {HTMLMediaElement} options.mediaElement
    *
-   * @see Refer to the <a href="https://github.com/bbc/audiowaveform/blob/master/doc/DataFormat.md">data format documentation</a>
+   * @see Refer to the <a href="../../../doc/DataFormat.md">data format documentation</a>
```

Comment:
Oh my bad! Because I was switching between the two repo I thought it was written as an absolute link and could be relative instead. Will rectify that :-)